client/src/Routes.jsx

```diff
@@ -1,28 +1,14 @@
 import React from 'react';
```

Comment:
What's the reasoning behind separating the React Routers and the components? Was it strictly separation of concern?

Reply:
Yeah I had the main component in the file originally, but moved it into Layout.jsx as it became more detailed